libs/vkd3d/device.c

```diff
@@ -2949,6 +3155,8 @@ static HRESULT vkd3d_create_vk_device(struct d3d12_device *device,
     device_info.ppEnabledExtensionNames = extensions;
     device_info.pEnabledFeatures = &device->device_info.features2.features;
     vkd3d_free(user_extension_supported);
+    vkd3d_free(openvr_extensions);
```

Comment:
This seems problematic. vkd3d_enable_extensions does not seem to strdup the extensions.